types/mapcmd/map.go

```diff
@@ -48,6 +75,35 @@ func getMap(ctx context.Context, clientConfig *hazelcast.Config, mapName string)
 	return
 }
 
+func NewUse() *cobra.Command {
+	//var mapName, mapKey string
+	cmd := &cobra.Command{
+		Use:   "use map-name",
+		Short: "sets default map name",
+		Example: "map use m1		# sets the default map name to m1 unless set explicitly\n" +
```

Comment:
`Use` and `Example` don't match.

Reply:
Maybe we can add `--reset` usage to the `Use` as well like `use map-name | use --reset`. Did you say they don't match because of this?

Reply:
@srknzl No. "Use" starts with `use map-name` but "Example" starts with `map use`.

Reply:
Examples include the `map` prefix where `Use` fields don't include them. I think it is intentional.

Reply:
Yes as @srknzl mentioned, cobra prefixes parent command names to "use" section.

Reply:
@srknzl I have added reset flag to "Use" field on https://github.com/hazelcast/hazelcast-commandline-client/pull/61/commits/808431d03755b7dda7649c1176a911453b160d31